netmiko/extreme/extreme_exos.py

```diff
@@ -53,21 +55,26 @@ def send_command(self, *args, **kwargs):
         self.set_base_prompt()
         return super().send_command(*args, **kwargs)
 
-    def config_mode(self, config_command=""):
+    def config_mode(self, config_command: str = "", *args: Any, **kwargs: Any) -> str:
         """No configuration mode on Extreme Exos."""
         return ""
 
-    def check_config_mode(self, check_string="#"):
+    def check_config_mode(
+        self, check_string: str = "#", *args: Any, **kwargs: Any
+    ) -> bool:
```

Comment:
Probably just define it as:

```python
def check_config_mode(self, check_string: str = "#", pattern: str = "") -> bool:
```

And then add `pattern=pattern` in the super() call